docs/usage_guide/examples/index.rst

```diff
@@ -0,0 +1,21 @@
+Modin Usage Examples
+====================
+
+This section shows Modin usage examples in different scenarios like Modin on a local/remote cluster,
+Modin in the cloud, the use of Modin spreadsheet and whatnot. First of all you may want to look at the basic
```

Comment:
Might be useful to add links to the spreadsheet API doc here, and if we have other links in the doc for installation guides.

Reply:
The links for the spreadsheet API and other topics are not presented here is because they're located in different directories for some reason. Spreadsheet API and all the other stuff is stored at the [`/docs/advanced_usage`](https://github.com/YarShev/modin/tree/dev/yigoshev-issue3863/docs/advanced_usage) when the getting started guide is located at the [`/docs/usage_guide`](https://github.com/YarShev/modin/tree/dev/yigoshev-issue3863/docs/usage_guide). For me it seems like duplication and I would combine these two sections. 

@YarShev was this section split intentional or we can combine them to reduce duplication with no losses?

Reply:
I suggest moving advanced_usage in usage_guide.

Reply:
combined the sections